.github/workflows/test.yml

```diff
@@ -0,0 +1,36 @@
+name: test
```

Comment:
We already have CI setup. We use Azure pipelines

Reply:
Yep, am not intending this to be merged. Still figuring out the workflow so I can run tests on my own forks.